aspnetcore/grpc/why-migrate-wcf-to-dotnet-grpc.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Why migrate WCF to ASP.NET Core gRPC
+author: markrendle
+description: A summary of why ASP.NET Core gRPC is a good fit for WCF developers who want to migrate to modern architectures and platforms.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: wpickett
+ms.date: 09/02/2020
+no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: grpc/wcf
+---
+
+# Why we recommend gRPC for WCF developers
+
+This article provides a summary outlining why ASP.NET Core gRPC is a good fit for WCF developers who want to migrate to modern architectures and platforms.
+
+## Similarity to WCF
+
+Although the implementation and approach are different for gRPC, the experience of developing and consuming services with gRPC should be intuitive for WCF developers. The underlying goal is the same: make it possible to code as though the client and server are on the same platform, without needing to worry about networking.
+
+Both platforms share the principle of declaring and then implementing an interface, even though the process for declaring that interface is different. And as you'll see in chapter 5, the different types of RPC calls that gRPC supports map well to the bindings available to WCF services.
```

Comment:
"And as you'll see in chapter 5" this doesn't mean anything to the reader. They haven't been told about the book yet.